### PR TITLE
added transaction verification tests, removed a couple node tests, sm…

### PR DESCRIPTION
**Pull request by doigensan**
…all import fix local IDE was complaining about.

### CODE REVIEW

File: test/integration/transactions.ts [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+            const resultFromExplorer: { data: { txid: string } } = await axios.get(`${ params.explorerUrl }/tx/${ validTx }`);
+
+            const txBytesFromNode: { data: { result: { transactionBytes: string } } } =
+                await axios.post(`${ params.nodeUrl }/transaction/findById`, { transactionId: validTx, blockHash: validBlockHash });
+
+            const txDecodeFromBytes: { data: { result: { transaction: { id: string } } } } =
+                await axios.post(`${ params.nodeUrl }/transaction/decodeTransactionBytes`, { transactionBytes: txBytesFromNode.data.result.transactionBytes });
+
+            expect(resultFromExplorer.data.txid).to.equal(txDecodeFromBytes.data.result.transaction.id);
```
Comment (otoumas) [inline test/integration/transactions.ts:23]: Would it be possible to compare more fields of the transaction? For instance we could compare the blockhash, the number of boxes (input and output), the public key of the boxes? dates?

**Review by otoumas — APPROVED** [review]
LGTM! Nice job! I added one comment

File: test/integration/transactions.ts (hunk 1 above)
Comment (carlosper88) [reply test/integration/transactions.ts:23]: Are we sure we want to hardcode hashes in the tests? Can this be an issue if eventually they're run over a different set of data (another node)? Maybe this is a dumb comment, but that's my concern about hardcoding in tests. Maybe we can make use of the endpoint "/blocks" that will return the list of the latest blocks with the transactions ids within

**Review by carlosper88 — COMMENTED** [review]
I leave a comment. It's more a doubt than an actual requirement. Looks great!

File: test/integration/transactions.ts (hunk 1 above)
Comment (doigensan) [reply test/integration/transactions.ts:23]: @otoumas a lot of the metadata is only available from the explorer node side (blockHash, blockTime, blockHeight) but the input/outputs could definitely use some comparisons!

@carlosper88 good suggestion, no reason these tests need to have hardcoded hashes/txs.  Just made them dynamic.

**Review by carlosper88 — APPROVED** [review]
Great job!

**Review by otoumas — APPROVED** [review]
LGTM

**Pull request merged**